trinity/utils/xdg.py

```diff
@@ -0,0 +1,63 @@
+import os
+
+
+XDG_CACHE_HOME = os.environ.get(
```

Comment:
What is XDG? The only things I came up with are Dogecoin and X Desktop group.

Reply:
A standard to specify home directory and similar, where to look for it etc... https://specifications.freedesktop.org/basedir-spec/basedir-spec-0.6.html

Reply:
See https://github.com/ethereum/py-evm/issues/198